modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -588,6 +597,12 @@ impl ServerModule for Wallet {
 
                     dbtx.remove_entry(&PegOutTxSignatureCI(txid)).await;
                     dbtx.remove_entry(&UnsignedTransactionKey(txid)).await;
+                    let propose_citem = self.propose_citem.clone();
+                    let broadcast_pending = self.broadcast_pending.clone();
+                    dbtx.on_commit(move || {
```

Comment:
I don't think it's strictly necessary to broadcast here when these keys are removed,but  with notifications it doesn't hurt to over-notify when in doubt.